parm/parm_fv3diag/field_table.aero

```diff
@@ -1,127 +1,127 @@
-# prognostic chemistry tracers
+# prognostic aerosols
```

Comment:
@WalterKolczynski-NOAA @rmontuoro @bbakernoaa  This was updated to match the field_table_thompson_noaero_tke_GOCART part for aerosols and the field_table_thompson_noaero_tke_GOCART file was deleted.  If there is any concern about this please let me know.

Reply:
My only concern is whether these values are appropriate if we use GFDL MP. @rmontuoro @bbakernoaa, can you weigh in on whether these are either okay for backwards-compatibility or we don't have to worry about running aerosol with GFDL MP anymore?

Reply:
I thought this was the solution you wanted? I can revert my last change but then we'll need to change the way this is handled in the workflow.

Reply:
It is if we can pull it off. All those zeros just have me nervous. Yes, if these aren't valid we will have to go back to the other way of doing it with one monolithic field_table. I'll take care of those modifications if we need to go there.

Reply:
Okay, we'll wait for word from @rmontuoro and @bbakernoaa.  I copied the field table from what is being used in the regression tests set-up.  If this needs to be changed based on what the aerosol model settings, then it is also incorrect in ufs-weather-model and needs to be updated there as well.

Reply:
@WalterKolczynski-NOAA, @JessicaMeixner-NOAA - The `parm/parm_fv3diag/field_table.aero` is independent of MP and looks correct.

@WalterKolczynski-NOAA - were are the zeros coming from?

Reply:
@rmontuoro The changes Jessica made change the surface value of all the fields to 0.0, presumably to match what is in the [Thompson TKE field table in UFS](https://github.com/ufs-community/ufs-weather-model/blob/develop/tests/parm/field_table/field_table_thompson_aero_tke)

Reply:
@WalterKolczynski-NOAA - These changes look good. All chemistry tracers are initialized to zero since they get populated through ICs.

Reply:
Sounds like this is fine then